Python/structmember.c

```diff
@@ -4,6 +4,26 @@
 #include "Python.h"
 #include "pycore_abstract.h"      // _PyNumber_Index()
 #include "pycore_long.h"          // _PyLong_IsNegative()
+#include "pycore_pyatomic_ft_wrappers.h"
+#include "pycore_object.h"
+#include "pycore_critical_section.h"
+
+
+static inline PyObject *
+_PyMember_GetOneObject(const char *addr, const char *obj_addr, PyMemberDef *l)
```

Comment:
Static functions in `.c` files don't usually use a `Py` prefix. Something like `member_get_object` is fine.